openssl.yaml

```diff
@@ -60,7 +52,10 @@ pipeline:
          no-ssl3 \
          no-seed \
          no-weak-ssl-ciphers \
+         no-des \
+         no-dsa \
          -Wa,--noexecstack
+      perl configdata.pm --dump
```

Comment:
Just checking that this is not debug stuff left around since it wasn't there before.

Reply:
Intentional commit and mentioned in the pull request description.

The output this produces allows inspection of all the things enabled and disabled. After configure is executed it even suggests to run that to see the configuration state.
Many keys are non declarative, and cascade enable/disable things in unexpected ways.